samples/README.md

```diff
@@ -21,75 +21,19 @@ library's README.
 
 ## Samples
 
-### Logging
+### Quickstart
 
-View the [source code][logs_0_code].
+View the [source code][quickstart_0_code].
```

Comment:
The preview of this part under samples is a little weird. Could you please verify the layout in a correct format?

Reply:
Fixed.